### ISSUE TITLE
Pitches up with tailwind on ground

### ISSUE THREAD

**Issue by dsatgh** [HHS81/c182s#517]
When I'm on the ground with a tailwind somewhere above 10 kts, the plane pitches up until the tail hits the ground. I noticed a big difference in behavior to the c172p. I don't know if it's relevant, but when I look at the wind pitch moment in the properties, the c172p show much smaller values and reacts to elevator movements. The c182s has a much bigger value and doesn't react much to elevator movements.

**Comment by HHS81** [comment]
Did you check that the aircraft AND FlightGear version is the most recent?

I just tried, and yes, the aircraft is pitching up with winds more than 15ktn. The values should have been realistic once I set them, but may be altered some time ago. I'll check that

**Comment by hbeni** [comment]
Also look at the CoG: how was the plane loaded when this occurred? (Tanks, Passengers, Baggage)

**Comment by dsatgh** [comment]
I'm on master branch and put the CoG as much forward as possible.

**Comment by HHS81** [comment]
> Also look at the CoG: how was the plane loaded when this occurred? (Tanks, Passengers, Baggage)

I could reproduce it. The first impression is comparing different numbers, that it is not the elevator- there is something else.

**Comment by hbeni** [comment]
Any new insights into this? :pray:

**Comment by HHS81** [comment]
> Any new insights into this? 🙏

Not yet, at least I can say it is not related with the climb rates issue. I do know that the C182 is sensible to wind on the ground and therefore it will need correct inputs of the yoke wheel in order not to roll over while taxiing, But here there inputs doesn`t have the needed effect. I have to try different things to rule certain things out.

**Comment by hbeni** [comment]
Interesting read, maybe related, on the Forums:

https://forum.flightgear.org/viewtopic.php?p=417356#p417356

> It doesn't actually take much to tip a C172 or PA28 onto its tail; if you go to an airport after a heavy snowfall, you'll find several planes with their nosewheels in the air

and also that the fdm models gear is not spot on the 3D-model ones for the 172, something we should ceck here too

**Comment by HHS81** [comment]
Unless something had been changed on the fdm since I started the model, the fdm gears should be spot on. That was something I checked really early on the model.

**Comment by hbeni** [comment]
Had the mood to poke around with versions a bit to try to get a grip on when this was introduced.
All tests performed with:
* max forward CoG (pilot+copilot full weight, full tanks, otherwise no passengers or baggage) and elevator full down.
* Weather was set to 'manual' in Launcher and `XXXX 012345Z 09010KT 9999 -SN OVC060CB SCT050TCU 20/M30 Q1005`
* location was EGNX, runway 27 (threshold)

----

* **FGFS 2020.3.19:**
** with current `master` (f23bf4ffc717d916bc7) pitched up with 13kt; CoG=38.94.
That seems to be pretty much the tipping point: more backward CoG pitches up and more wind also; moving the elevator to neutral also. ( :partying_face: btw nice that it still loads on stable and is flyable, good to know that)
** `version-1.2` tag (10f071fb5920b81) from Jul 31 19:23:18 2016: sustained till 55kts CoG however was at 39.00 (so there is a small difference already)!

So that does in principal verify it is indeed not flightgear itself causing this, but FDM related.
I continued to test on next.

----
* **FGFS next:**
** current `master` same behaviour as in 2020.3.19.
** `version-1.2` same behaviour as in 2020.3.19.
** `version-1.4` has again the problematic behaviour (CoG=38.95), pitch up at 14kts.

---- 
So... I `git bisect`ed' from that on _next_ and got:
> 7a5a0b60868bf7f219b9f51467d459031a91fef0 is the first bad commit

Verified that by reverting it at masters Head (resolving the conflicts), and indeed the problem went away.
Pushed branch `Issue_517-TailwindPitchUp` for investigation; its `master` with 7a5a0b6086 reverted.

**Comment by HHS81** [comment]
@hbeni 
So it was me who introduced the bug.
With the changing on the gear  I also changed some values on the pitch coefficient- but can't remember why. I'll check that immediately, so we can revert the numbers on pitch coeffcients.

**Comment by hbeni** [comment]
That would be awesome;
I honestly have no clue what those numbers really mean, so I stopped trying to fix it...

**Comment by hbeni** [comment]
Can the pitch coefficient also introduce more lift?
Just asking again because of the climb numbers

**Comment by HHS81** [comment]
> That would be awesome; I honestly have no clue what those numbers really mean, so I stopped trying to fix it...

I testet it and it works for me flawless. I guess I might have changed it, as there had been complaints about to be too sensitive on the elevator controls. But I remember some cockpit videos from real aircraft, which showed it needs only little input to takeoff

**Comment by HHS81** [comment]
> Can the pitch coefficient also introduce more lift? Just asking again because of the climb numbers

No, and the climb numbers stayed the same with. I might have some time on the weekend, to finally correct this issue.

**Comment by HHS81** [comment]
@hbeni I will push/ pull request do tomorrow or saturday, as I need to update my authentification.

**Comment by HHS81** [comment]
As I said- the pitch up with tailwind is also a problem on the c172p (winking to @wlbragg ) and was there introduced with the ommit 3680740 while changing the adverse yaw effect.

**Comment by hbeni** [comment]
See also: https://github.com/c172p-team/c172p/issues/1489
There are also linked forum topics

**Comment by ysopflying** [comment]
The formula without taking propwash into account and using a simple sin-expression can certainly be improved.

**Comment by ysopflying** [comment]
current FGAddon stable has:
```
            <function name="aero/coefficient/Cmalpha">
                <description>Pitch_moment_due_to_alpha</description>
                <product>
                    <property>aero/qbar-psf</property>
                    <property>metrics/Sw-sqft</property>
                    <property>metrics/cbarw-ft</property>
                    <sin>
                        <property>aero/alpha-rad</property>
                    </sin>
                    <value>-1.8</value> 
```
The sin-expression is not present on current github:
```

                    <property>metrics/cbarw-ft</property>

                        <property>aero/alpha-rad</property>

                    <value>-1.65</value> 
```
If there is a high negative alpha due to tailwind, a high pitch up angle occurs.

**Comment by hbeni** [comment]
And what is the correct fix for the Situation?

My programmers mind just wants to add clamping to high negative alpha due to tailwind… which is certainly an ugly, if not evil, hack.

**Comment by TheFGFSEagle** [comment]
The non-ugly fix would be to use the first code, instead of the second in Ysops last post ! ;)

**Comment by hbeni** [comment]
> current FGAddon stable has

Looks like it was commit b8e32b70710d651872a

**Comment by ysopflying** [comment]
From a simple NACA0012 (using factors for 3D, different thickness, neglecting fuselage, main wing.....):

<body>

Alpha wing | cmm
-- | --
-180 | -0.24
-175.5 | 0
-171.305 | 0.23
… | …
174.659 | -0.43
176.575 | -0.39
180.305 | -0.23


</body>
Actually it would need propwash effect. To be done at a later stage.
Excluded are all other angles, as there are different versions floating around in different commits. 
Alpha -10-25 might be subject to a separate issue.

**Comment by hbeni** [comment]
Thanks, but where does this need to go (code wise)?

**Comment by ysopflying** [comment]
will be completed to a full commit after having a look at a few things

**Comment by ysopflying** [comment]
![Screenshot from 2024-04-21 11-07-54](https://github.com/HHS81/c182s/assets/91372156/c252c711-30e8-4fbc-92bf-e8dc0b3cf93b)


see https://github.com/HHS81/c182s/commit/f76709d3ec108acff56726f2e736458080b36289
see https://github.com/ysopflying/c182s/commit/58ada8a08f74e321625b5f7cae6815bad3558d61
